js/webviews.js

```diff
@@ -224,7 +224,8 @@ const webviews = {
           allowPopups: false,
           partition: partition,
           enableWebSQL: false,
-          autoplayPolicy: 'user-gesture-required'
+          autoplayPolicy: 'user-gesture-required',
+          navigateOnDragDrop: true
```

Comment:
Is this actually useful? The only times I remember encountering this is trying to drag a file onto a page to upload it, dragging it to the wrong spot, and accidentally navigating instead. If you think it's useful, I'm fine with adding it, but if no one is specifically requesting it I'd probably leave it out.